### PR TITLE
Combine consecutive unsets

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | /
| Type?         | improvement
| Category?     | CO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | /
| How to test?  | /

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11697)

### CODE REVIEW

File: classes/Cookie.php [hunk 1]
```diff
@@ -259,25 +259,15 @@ public function logout()
...
-        unset($this->_content['id_connections']);
-        unset($this->_content['customer_lastname']);
-        unset($this->_content['customer_firstname']);
-        unset($this->_content['passwd']);
-        unset($this->_content['logged']);
-        unset($this->_content['email']);
-        unset($this->_content['id_cart']);
-        unset($this->_content['id_address_invoice']);
-        unset($this->_content['id_address_delivery']);
+        unset($this->_content['id_customer'], $this->_content['id_guest'], $this->_content['is_guest'], $this->_content['id_connections'], $this->_content['customer_lastname'], $this->_content['customer_firstname'], $this->_content['passwd'], $this->_content['logged'], $this->_content['email'], $this->_content['id_cart'], $this->_content['id_address_invoice'], $this->_content['id_address_delivery']);
```
Comment (PierreRambaud) [inline classes/Cookie.php]: It's not readable :/

File: controllers/admin/AdminRequestSqlController.php [hunk 2]
```diff
@@ -195,10 +195,7 @@ public function ajaxProcess()
             $request_sql = new RequestSql();
             $attributes = $request_sql->getAttributesByTable($table);
             foreach ($attributes as $key => $attribute) {
-                unset($attributes[$key]['Null']);
-                unset($attributes[$key]['Key']);
-                unset($attributes[$key]['Default']);
-                unset($attributes[$key]['Extra']);
+                unset($attributes[$key]['Null'], $attributes[$key]['Key'], $attributes[$key]['Default'], $attributes[$key]['Extra']);
```
Comment (PierreRambaud) [inline controllers/admin/AdminRequestSqlController.php]: Same here

File: tests-legacy/resources/ModulesOverrideInstallUninstallTest/AdminProductsController.php [hunk 3]
```diff
@@ -2318,8 +2318,8 @@ public function initContent($token = null)
             if (!$id_category) {
                 $this->_defaultOrderBy = $this->identifier;
                 if ($this->context->cookie->{$this->table.'Orderby'} == 'position') {
-                    unset($this->context->cookie->{$this->table.'Orderby'});
-                    unset($this->context->cookie->{$this->table.'Orderway'});
+                    unset($this->context->cookie->{$this->table.'Orderby'}, $this->context->cookie->{$this->table.'Orderway'});
```
Comment (PierreRambaud) [inline tests-legacy/resources/ModulesOverrideInstallUninstallTest/AdminProductsController.php]: same here

File: tests-legacy/resources/module/pscsx32412/override/controllers/admin/AdminProductsController.php [hunk 4]
```diff
@@ -2132,8 +2132,8 @@ public function initContent($token = null)
             if (!$id_category) {
                 $this->_defaultOrderBy = $this->identifier;
                 if ($this->context->cookie->{$this->table.'Orderby'} == 'position') {
-                    unset($this->context->cookie->{$this->table.'Orderby'});
-                    unset($this->context->cookie->{$this->table.'Orderway'});
+                    unset($this->context->cookie->{$this->table.'Orderby'}, $this->context->cookie->{$this->table.'Orderway'});
```
Comment (PierreRambaud) [inline tests-legacy/resources/module/pscsx32412/override/controllers/admin/AdminProductsController.php]: same here

**Review by PierreRambaud — DISMISSED** [review]
Break lines and it will be ok :)

**Comment by MathiasReker** [comment]
@PierreRambaud Sure, I go afk. But I will fix it tomorrow.

**Comment by MathiasReker** [comment]
Sorry for commit spam, I used the web editor xD

**Review by jolelievre — APPROVED** [review]

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
I squashed the commits together. Thank you @MathiasReker

**Pull request merged**